mock-core-configs/etc/mock/eol/templates/mageia-7.tpl

```diff
@@ -30,9 +31,10 @@ user_agent={{ user_agent }}
 
 [mageia]
 name=Mageia $releasever - {{ target_arch }}
-#baseurl=http://mirrors.kernel.org/mageia/distrib/$releasever/{{ target_arch }}/media/core/release/
+# Mirrorlist doesn't work ATM as old Mageia repos are only available in distrib-coffee's Mageia-archive
+baseurl=https://distrib-coffee.ipsl.jussieu.fr/pub/linux/Mageia-archive/distrib/$releasever/{{ target_arch }}/media/core/release/
```

Comment:
They at least appear to be working :) what am I missing?
```
curl 'https://www.mageia.org/mirrorlist/?release=9&arch=x86_64&section=core&repo=release'
ftp://ftp.fi.muni.cz/pub/linux/mageia/distrib/9/x86_64/media/core/release/
ftp://vodka.sublink.org/mageia/distrib/9/x86_64/media/core/release/
http://ftp.cc.uoc.gr/pub/linux/mageia/distrib/9/x86_64/media/core/release/
http://fr2.rpmfind.net/linux/mageia/distrib/9/x86_64/media/core/release/
ftp://mirror.datacenter.by/pub/mirrors/mageia.org/distrib/9/x86_64/media/core/release/
ftp://mirrors.netix.net/mageia/distrib/9/x86_64/media/core/release/
http://quantum-mirror.hu/mirrors/pub/mageia/distrib/9/x86_64/media/core/release/
http://ftp.klid.dk/ftp/mageia/distrib/9/x86_64/media/core/release/
ftp://mirror2.tuxinator.org/mageia/distrib/9/x86_64/media/core/release/
ftp://ftp.snt.utwente.nl/pub/os/linux/mageia/distrib/9/x86_64/media/core/release/
```

Reply:
Note that using a single mirror in default mock-core-configs is a bad idea;  it will both cause DDoS on the mirror and make the builds slow for far distances.

Reply:
Then we should have the mirrorlist service return just that one URL for archived releases.

Reply:
> ATM mirrorlist doesn't handle correctly archived Mageia releases.

`curl 'https://www.mageia.org/mirrorlist/?release=7&arch=x86_64&section=core&repo=release'`
returns only non-working entries.